src/configuration/app-containers.md

```diff
@@ -116,11 +119,15 @@ The `type` can be:
 
 The `build` defines what will happen by default when building the application.
 
-It has a sub property `flavor` for which the possible values are:
+Its only property is `flavor`, which specifies a default set of build tasks to run. Flavors are language-specific.
+
+* PHP - default `composer`
+  * `composer` will run `composer --no-ansi --no-interaction install --no-progress --prefer-dist --optimize-autoloader` if a `composer.json` file is detected.
+  * `drupal` will run `drush make` automatically if a `.make` file is detected.
+* Node.js
+  * `default` will run `npm prune --userconfig && npm install --userconfig` if a `package.json` file is detected.
```

Comment:
Maybe just `npm prune && npm install`, because the --userconfig is actually `--userconfig <somefilename>` and it's probably a minor implementation detail.